src/pudl/io_managers.py

```diff
@@ -477,6 +509,21 @@ def _handle_pandas_output(self, context: OutputContext, df: pd.DataFrame):
 
         df = res.enforce_schema(df)
 
+        if self.settings.write_to_parquet:
+            logger.info(f"Writing {self.db_name}/{table_name} to parquet.")
+            parquet_path = PudlPaths().parquet_path(self.db_name, table_name)
+            parquet_path.parent.mkdir(parents=True, exist_ok=True)
+            schema = res.to_pyarrow()
```

Comment:
Along with the schema, another resource-specific parameter we might want in here is information about how to organize the row groups for larger assets, which I guess could be another method we add to `Resource`

Reply:
This doesn't currently exist. The only big table we've written out to Parquet historically is the EPA CEMS hourly emissions, and that's been done in a bespoke way.

Reply:
Yeah, I think we can encode this in the resource, or in the asset metadata once we're ready for it. The code should be flexible enough to accommodate that once we need it.